CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - TBD
+### Added
+- Add `LifecycleEventType` enum in `beekeeper-core` to describe supported data Lifecycles.
+- Increase unit test coverage for `beekeeper-path-scheduler-apiary` package.
```

Comment:
Remove this and the next line, we generally don't mention changes to tests in the CHANGELOG as it's usually not that interesting. Most people reading it are downstream devs/users who are just interested in the changes to the application itself.